src/main/resources/default/assets/tycho/scripts/charts.js.pasta

```diff
@@ -345,3 +345,19 @@ function barChart(selector, labels, datasets) {
         options: customConfig
     });
 }
+
+function dougnutChart(selector, labels, data) {
```

Comment:
The name of the function is less discriptive. :)

Reply:
This is in style of the other chart functions.

Reply:
okay